airbyte-cdk/bulk/core/load/build.gradle

```diff
@@ -37,9 +37,12 @@ task integrationTest(type: Test) {
     maxParallelForks = project.test.maxParallelForks
     maxHeapSize = project.test.maxHeapSize
 }
+// These tests are lightweight enough to run on every PR.
+tasks.named('check').configure {
+    dependsOn integrationTest
+}
```

Comment:
this makes `:integrationTest` run when `:check` is run which happens when `:build` is run therefore `:bulkCdkBuild` will include this